site/en/blog/deprecating-unload/index.md

```diff
@@ -54,6 +54,8 @@ Instead of `unload` it is recommended to use:
 - [`visibilitychange`](/articles/page-lifecycle-api/#event-visibilitychange): To determine when the visibility of a page changes. This event happens when the user switches tabs, minimizes the browser window, or opens a new page. Consider the [`hidden` state](/articles/page-lifecycle-api/#advice-hidden) the last reliable time to save app and user data.
 - [`pagehide`](/articles/page-lifecycle-api/#event-pagehide): To determine when the user has navigated away from the page. This event happens when the user navigates away from the page, reloads the page, or closes the browser window. The `pagehide` event is not fired when the page is simply minimized or switched to another tab. Note that, as `pagehide` does not make a page ineligible for the back/forward cache, it is possible a page can be restored after this event fires. If you're cleaning up any resources in this event, then they may have to be restored on page restore.
 
+The `beforeunload` event has a slightly different use case to `unload` in that it is a cancellable event. It is often use to warn users of unsaved data when navigating away. This event is also unrealiable as it will not fire if a background tab is killed. It is recommended to limit use of `beforeunload` and [only add this conditionally](/blog/page-lifecycle-api/#the-beforeunload-event), and instead use the above events for most `unload` replacements.
```

Comment:
```suggestion
The [`beforeunload`](https://developer.mozilla.org/docs/Web/API/Window/beforeunload_event) event has a slightly different use case to `unload` in that it is a cancellable event. It is often used to warn users of unsaved changes when navigating away. This event is also unrealiable as it will not fire if a background tab is killed. It is recommended to limit use of `beforeunload` and [only add it conditionally](/blog/page-lifecycle-api/#the-beforeunload-event). Instead, use the above events for most `unload` replacements.
```